pydal/adapters/base.py

```diff
@@ -1880,7 +1880,7 @@ def execute_test_query(self):
         '''
         return None
 
-    def represent(self, obj, fieldtype):
+    def represent(self, obj, fieldtype, object_id=long):
```

Comment:
I'm not sure is good to have and `object_id` parameter here which will be used only with Mongo.
Probably I will merge this after **15.07** release and then refactor the code a bit.